### DISCUSSION TITLE
Would you recommend any specific gunicorn settings for a feed?

### DISCUSSION THREAD

**Discussion by venteto** [MarshalX/atproto#223] [Questions] [answered]
Every time I try to manually run gunicorn against my test feed, it dies about 30 seconds.

I'm not sure if a standardized gunicorn config file would work for every feed, or the configuration would really be dependent on whatever a specific feed is doing.

If I go ahead and deploy it to Fly with the Dockerfile using "flask run" as the CMD though it works, I'd just prefer to make it work with gunicorn.

This is my stub of a gunicorn config file:

```python
workers = 2
# threads = 4
# timeout = 120
bind = '0.0.0.0:8080'

# forwarded_allow_ips = '*'
# secure_scheme_headers = { 'X-Forwarded-Proto': 'https' }
```

**Comment by MarshalX** [comment]
Did you separate firehose consumer from web server? The original feed examples uses thread to run firehose consumer inside flask app

**Reply by venteto** [reply]
No, I haven't figured out how to do that yet. I've never used Flask before. But for the time being, while I am still experimenting, `flask run` is fine.

I think something that is possibly a higher priority to me to handle first is #224.

**Comment by MarshalX** [comment, marked as answer]
You can use this repository as an example https://github.com/bluesky-astronomy/astronomy-feeds. It's a good, popular, and stable custom feed written using atproto Python SDK. It uses gunicorn and run the firehose consumer separately
